ci-operator/step-registry/dpu-operator/e2e-tests-workflow/tests/dpu-operator-e2e-tests-workflow-tests-commands.sh

```diff
@@ -1,86 +1,70 @@
 #!/bin/bash
 
-echoerr() { echo "$@" 1>&2; }
+set -e
 
-_curl_qm() {
-    curl --resolve "$endpoint_resolve" --cacert "$queue_manager_tls_crt" -H "Authorization: Bearer $queue_manager_auth_token" "$@"
+echoerr() {
+    printf '%s\n' "$*" 1>&2
 }
 
-check_timeout_status() {
-    _curl_qm -X POST "$queue_url/check_timed_out_job" -H "Content-Type: application/json" -d "{\"pull_pull_sha\": \"${PULL_PULL_SHA}\"}"
+_curl_qm() {
+    curl -s --resolve "$endpoint_resolve" --cacert "$queue_manager_tls_crt" -H "Authorization: Bearer $queue_manager_auth_token" "$@" 2>/dev/null
 }
 
-trigger_build() {
-    _curl_qm -X POST "$queue_url" -H "Content-Type: application/json" -d "{\"pullnumber\": \"${PULL_NUMBER}\", \"pull_pull_sha\": \"${PULL_PULL_SHA}\"}"
+_json_get() {
+    printf '%s' "$1" | jq -r "$2"
```

Comment:
Aren't you missing the current command. This is just printing?

Reply:
The curl command is called previously.

The caller then calls `_json_get` (potentially multiple times), to extract values from the JSON.